src/main/java/uk/gov/companieshouse/api/strikeoffobjections/authorization/AttachmentDownloadAuthorizationInterceptor.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.companieshouse.api.strikeoffobjections.authorization;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+
+import uk.gov.companieshouse.api.strikeoffobjections.common.ApiLogger;
+import uk.gov.companieshouse.api.strikeoffobjections.service.impl.ERICHeaderFields;
+import uk.gov.companieshouse.service.ServiceException;
+
+public class AttachmentDownloadAuthorizationInterceptor extends HandlerInterceptorAdapter {
+
+    /**
+     * The admin role that is assigned to CHS users who are allowed to download objection attachments.
+     */
+    private static final String ADMIN_DOWNLOAD_ROLE = "/admin/strike-off-objections-download";
```

Comment:
would it be better to to have the role in config, it's a private repo

Reply:
Discussion on-going...

Reply:
leave them as they are, according to https://www.gov.uk/government/publications/open-source-guidance/when-code-should-be-open-or-closed  they are ok